tests/backup/backup_restore_basic_test.go

```diff
@@ -1318,3 +1318,156 @@ var _ = Describe("{BackupSyncBasicTest}", func() {
 		CleanupCloudSettingsAndClusters(backupLocationMap, credName, cloudCredUID, ctx)
 	})
 })
+
+// BackupMultipleNsWithSameLabel Backup and restore for single, multiple namespaces with same labels
+var _ = Describe("{BackupMultipleNsWithSameLabel}", func() {
+	var (
+		err                         error
+		backupLocationUID           string
+		cloudCredUID                string
+		clusterUid                  string
+		backupName                  string
+		credName                    string
+		restoreName                 string
+		backupLocationName          string
+		multipleNamespaceBackupName string
+		labelForSingleNamespace     string
+		labelForMultipleNamespace   string
+		restoreNames                []string
+		bkpNamespaces               []string
+		cloudCredUidList            []string
+		nsLabelsMap                 map[string]string
+		nsLabelsMapSingleNamespace  map[string]string
+		contexts                    []*scheduler.Context
+		appContexts                 []*scheduler.Context
+	)
+	labelSelectors := make(map[string]string)
+	backupLocationMap := make(map[string]string)
+	namespaceMapping := make(map[string]string)
+	bkpNamespaces = make([]string, 0)
+	JustBeforeEach(func() {
+		StartTorpedoTest("BackupMultipleNsWithSameLabel", "Verification of backup and restore for single, multiple namespaces with same labels", nil, 84851)
+		log.InfoD("Deploy applications")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < 2; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				bkpNamespaces = append(bkpNamespaces, namespace)
+			}
+		}
+		log.InfoD("Created namespaces %v", bkpNamespaces)
+	})
+	It("Verification of backup and restore for single, multiple namespaces with same labels", func() {
+		providers := getProviders()
+		Step("Validate applications", func() {
+			log.InfoD("Validate applications")
+			ValidateApplications(contexts)
+		})
+		Step("Adding labels to all namespaces", func() {
+			log.InfoD("Adding labels to all namespaces")
+			nsLabelsMap = GenerateRandomLabels(3)
+			err = AddLabelsToMultipleNamespaces(nsLabelsMap, bkpNamespaces)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Adding labels [%v] to namespaces [%v]", nsLabelsMap, bkpNamespaces))
+		})
+		Step("Adding labels to single namespace", func() {
```

Comment:
from teststeps , The idea of the testcase is to have a label applied to mutiple namespaces and create two set of backup's.
1. Select only one namespace from the filtered namespace with labels and create backup
2. Select all namespace filtered with label and create backup.

Reply:
we can not Select only one namespace from the filtered namespace with labels and create backup

Reply:
ok got it, In that case we need to update the steps in testrail , the steps there are misleading.